main/adapters/repository/database_init.go

```diff
@@ -1,28 +1,15 @@
 package repository
 
 import (
-	"context"
-	"database/sql"
 	"fmt"
 )
 
 const (
 	PostgresIdentity = iota
-	PostgresVersion
-	PostgresIdentityTableName  = "identity"
-	PostgresVersionTableName   = "version"
-	MigrationID                = "dbMigration"
-	MigrationVersionNoDB       = "0.0"
-	MigrationVersionInit       = "1.0.1"
-	MigrationVersionHashedAuth = "2.0"
-	MigrationVersionLatest     = "3.0"
-)
+	SQLiteIdentity
 
-type Migration struct {
-	id      string
-	version string
-	tx      *sql.Tx
-}
+	IdentityTableName = "identity"
+)
 
 var create = map[int]string{
 	PostgresIdentity: fmt.Sprintf("CREATE TABLE IF NOT EXISTS %s("+
```

Comment:
I like that the Database DDL Creation is embedded inside the binary. At one point in time, these need to be proper migrations. Otherwise this will be very hard to maintain.

We talked briefly about how to do this, and since there are a lot of differences between Postgres and SQLite, i think these should be two seperate migration collections, each part of the respective database.

Last time we decided to skip refactoring this for now, but I think we may have some low hanging fruits here.

Reply:
i am not sure what you mean, but I would be happy about a PR or a meeting, where you explain this to me. :) in case you want to do a PR, make sure you do it for the most current branch https://github.com/ubirch/ubirch-client-go/tree/feature/PROJ-105-support-external-identities.

Sorry about the mess. We are planning to merge everything this week.

Reply:
i will resolve this conversation as I don't see this as a blocking issue for the merge of this PR. Let me know if you disagree :)